R/zchunk_L2321.cement.R

```diff
@@ -1,20 +1,23 @@
 #' module_energy_L2321.cement
 #'
-#' Briefly describe what this chunk does.
+#' The chunk provides final energy keyword, supplysector/subsector information,
+#' supplysector/subsector interpolation information, global technology share weight,
+#' global technology efficiency, global technology coefficients, global technology cost,
+#' price elasticity, stub technology information, stub technology interpolation information,
+#' stub technology calibrated inputs, and etc for cement sector.
```

Comment:
I'd prefer this be shortened, and spelled out in full in `@details` below. E.g. "Compute a variety of final energy keyword, sector, share weight, and technology information for cement-related GCAM inputs."